wordel.el

```diff
@@ -311,6 +311,7 @@ If PROPS are non-nil, they are used in place of default values."
     \\{wordel-mode-map}"
   (add-hook 'pre-command-hook #'wordel--filter-inputs nil t)
   (setq header-line-format (wordel--commands-text))
+  (setq-local show-trailing-whitespace nil)
```

Comment:
Please combine this with the setq-local call above. It can take any number symbol/val pairs.